__tests__/app.test.js

```diff
@@ -109,3 +109,53 @@ describe.only("/api/reviews", () => {
     })
   });
 });
+
+describe("/api/reviews/:review_id/comments", () => {
+  it("GET 200: should respond with array of comments for review_id requetsed", () => {
+    return request(app)
+      .get("/api/reviews/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toBeInstanceOf(Array);
+        expect(body.comments).toBeSortedBy("created_at", {
+          descending: true,
+        });
+        expect(body.comments.length).toBe(3);
+        body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            review_id: expect.any(Number),
```

Comment:
As you've request id 3, you'd expect this to be 3, instead of any number. 

`review_id: 3`

At the moment, this would pass if you got the comments for a different review